src/include/ikarus/finiteElements/mechanics/linearElastic.hh

```diff
@@ -170,6 +171,27 @@ namespace Ikarus {
       }
     }
 
+    void calculateAt(const ResultRequirementsType& req, const Eigen::Vector<double, Traits::mydim>& local,
+                     ResultTypeMap<double>& result) const {
+      using namespace Dune::Indices;
+      using namespace Dune::DerivativeDirections;
+      using namespace Dune;
+
+      const auto eps = getStrainFunction(req.getFERequirements());
+      const auto C   = getMaterialTangent();
+      auto gp        = toDune(local);
+      auto epsVoigt  = eps.evaluate(gp, on(gridElement));
+
+      auto cauchyStress = C * epsVoigt;
+
+      typename ResultTypeMap<double>::ResultArray resultVector;
+      if (req.isResultRequested(ResultType::cauchyStress)) {
+        resultVector.resize(3, 1);
+        resultVector = cauchyStress;
```

Comment:
Like `resultVector = C * epsVoigt;`

Reply:
You can decide what feel more correct in the actual code

Reply:
I changed here:  `auto cauchyStress = (C * epsVoigt).eval();`